### DISCUSSION TITLE
controle sanitaire dans les shelters de guilde

### DISCUSSION THREAD

**Discussion by ghosts-arch** [Crownicles/Crownicles#1649] [Nouvelles idées]
des controlleurs qui viennent de manière régulière et qui mettent une amande pour chaque animal fielleux

**Comment by Procion06** [comment]
c'est une bonne idée de leurs donner des amandes, comme ça ils ne seront plus fielleux x)

**Comment by Akutana** [comment]
Je pense que le prix n'excéderait pas 1k sinon ça ferait beaucoup, sauf que ça reviendrait beaucoup moins cher de se prendre l'amende que ne serait-ce que maintenir un animal au moins sauvage
